CSS/Owner/elementHolderLogOut.css

```diff
@@ -0,0 +1,45 @@
+.elementHolder {
+    position: fixed;
+    top: 28%;
+    left: 35%;
+    right: 2%;
+    bottom: 2%;
+    border-radius: 40px;
+    background-color: floralwhite;
+    opacity: 0.6;
+}
+
+.elements {
+    position: absolute;
+    left: 10%;
+    top: 10%;
+    background-color: #541702;
+    color: floralwhite;
+    text-shadow: 0.5px 0.5px floralwhite;
+    font-size: medium;
+    opacity: +5;
+    font-weight: bold;
+    right: 10%;
+    bottom: 10%;
+    font-family: cursive;
+}
```

Comment:
have all the common classes in a single css file and use it in all your html files